docs/book/src/getting-started.md

```diff
@@ -90,9 +90,98 @@ The capability to set credentials using environment variables is now deprecated
 
 
 ### Building your first cluster
-Check out the [Cluster API Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.html) to create your first Kubernetes cluster on Azure using Cluster API. Make sure to select the "Azure" tabs.
```

Comment:
Add in comment at top of the page something like:  "This quickstart is for provisioning a managed (AKS) cluster with the CAPI-operator and helm.  See the CAPI quickstart to provision a self-managed cluster on Azure using `clusterctl`.  The identity instructions can apply to either self-managed or managed cluster provisioning."